DEPLOY.markdown

```diff
@@ -65,18 +66,12 @@ Be careful while collecting and releasing PSC results: the configured epsilon an
 
 PSC is not designed for automated collection and results release: a long enough series of results can identify the activity of a single user.
 
-After configuring parameters, run Gen_Config.go:
```

Comment:
The Tally Server needs to generate the configuration parameters before it can be run.